apps/books/migrations/0002_auto_20200811_1720.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.1 on 2020-08-11 12:20
+
+import datetime
+from django.db import migrations, models
+import django.utils.timezone
+from django.utils.timezone import utc
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('books', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='book',
+            name='created_at',
+            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='book',
+            name='updated_at',
+            field=models.DateTimeField(auto_now=True),
+        ),
+        migrations.AddField(
+            model_name='category',
+            name='created_at',
+            field=models.DateTimeField(auto_now_add=True, default=datetime.datetime(2020, 8, 11, 12, 20, 30, 130935, tzinfo=utc)),
```

Comment:
Please avoid giving default value through terminal.. Please add default in model as well